src/FreeType/FT2Library.class.st

```diff
@@ -92,8 +92,7 @@ FT2Library >> getBitmap: ftBitmap fromOutline: outline [
 
 { #category : #testing }
 FT2Library >> isAvailable [
-	self checkLibrary.
-	"^ false."
+	[self checkLibrary] on:Error do: [^ false].
```

Comment:
hmm, I do not think that to catch a general error is a good idea... it can hide some other issues

Reply:
The problem that handling less can hang up the startup.
The error can be FFI related or other stuff.

Reply:
hmm, at least we should log them somewhere later... probably good enough for now